### PR TITLE
Enable CRB on EL9 always

### PR DESCRIPTION
**Pull request by wbclark**

### CODE REVIEW

**Comment by evgeni** [comment]
That's needed for libvirt-devel being installable?

**Comment by wbclark** [comment]
> That's needed for libvirt-devel being installable?

Yes, exactly

**Comment by evgeni** [comment]
CI hates you (probably missing fixtures entry for yumrepo_core?), but otherwise ACK

**Comment by wbclark** [comment]
> CI hates you (probably missing fixtures entry for yumrepo_core?), but otherwise ACK

Yeah, trying to figure out why it was fine when we did it in setup_acceptance_node.pp but not here D:

**Comment by evgeni** [comment]
The failure is in the unit tests, which runs on the gem puppet that has no core modules. Acceptance runs AIO that has them.

File: metadata.json [hunk 1]
```diff
@@ -28,6 +28,10 @@
       "name": "puppetlabs-vcsrepo",
       "version_requirement": ">= 1.0.0"
     },
+    {
+      "name": "puppetlabs-yumrepo_core",
```
Comment (evgeni) [inline metadata.json:32]: That's not needed, IMHO

Comment (wbclark) [reply metadata.json:32]: ah, I see the explanation about gem puppet vs. AIO... I'll remove it in a moment

Thanks!

**Review by evgeni — APPROVED** [review]

**Pull request merged**